src/sentry/static/sentry/app/views/performance/vitalDetail/vitalPercents.tsx

```diff
@@ -1,44 +1,67 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
+import Tooltip from 'app/components/tooltip';
 import {IconCheckmark, IconFire, IconWarning} from 'app/icons';
-import {t} from 'app/locale';
+import {t, tct} from 'app/locale';
 import space from 'app/styles/space';
+import {WebVital} from 'app/utils/discover/fields';
 import {formatPercentage} from 'app/utils/formatters';
 import theme, {Color} from 'app/utils/theme';
 
-import {VitalState, vitalStateColors} from './utils';
+import {VitalState, vitalStateColors, webVitalMeh, webVitalPoor} from './utils';
 
 type Percent = {
   vitalState: VitalState;
   percent: number;
 };
 
 type Props = {
+  vital: WebVital | WebVital[];
   percents: Percent[];
   showVitalPercentNames?: boolean;
 };
 
+const vitalStateIcons = {
+  [VitalState.POOR]: IconFire,
+  [VitalState.MEH]: IconWarning,
+  [VitalState.GOOD]: IconCheckmark,
+};
+
+function vitalStateText(vital, vitalState) {
+  switch (vitalState) {
+    case VitalState.POOR:
+      return Array.isArray(vital)
+        ? t('Poor')
+        : tct('Poor: >[threshold]ms', webVitalPoor[vital]);
+    case VitalState.MEH:
+      return Array.isArray(vital)
+        ? t('Needs improvement')
+        : tct('Needs improvement: >[threshold]ms', webVitalMeh[vital]);
+    case VitalState.GOOD:
+      return Array.isArray(vital)
+        ? t('Good')
+        : tct('Good: <[threshold]ms', webVitalMeh[vital]);
+    default:
+      return null;
+  }
+}
+
 export default function VitalPercents(props: Props) {
   return (
     <VitalSet>
       {props.percents.map(p => {
+        const VitalStateIcon = vitalStateIcons[p.vitalState];
         return (
-          <VitalStatus key={p.vitalState}>
-            {p.vitalState === VitalState.POOR && (
-              <IconFire color={vitalStateColors[p.vitalState] as Color} />
-            )}
-            {p.vitalState === VitalState.MEH && (
-              <IconWarning color={vitalStateColors[p.vitalState] as Color} />
-            )}
-            {p.vitalState === VitalState.GOOD && (
-              <IconCheckmark color={vitalStateColors[p.vitalState] as Color} isCircled />
-            )}
-            <span>
-              {props.showVitalPercentNames && t(`${p.vitalState}`)}{' '}
-              {formatPercentage(p.percent, 0)}
-            </span>
-          </VitalStatus>
+          <Tooltip key={p.vitalState} title={vitalStateText(props.vital, p.vitalState)}>
+            <VitalStatus>
+              <VitalStateIcon color={vitalStateColors[p.vitalState] as Color} />
+              <span>
+                {props.showVitalPercentNames && t(`${p.vitalState}`)}{' '}
```

Comment:
I saw the thread in slack, but wanted to address it here for posterity: using `t()` with variables will function fine, but the translation part will do nothing.

Reply:
I'm virtually certain this is dead code. I have another PR lined up to delete showVitalPercentNames entirely, but it obviously merge-conflicts with this one.

Reply:
This is used [here](https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/performance/vitalDetail/vitalDetailContent.tsx#L183-L188), thought an argument can be made that with the tooltips, it's okay to remove this anyways.

Reply:
Ah, I see. I think it should be consistent everywhere anyways, and the tooltip is probably clear enough that we can delete it entirely.